tests/test_selenium.py

```diff
@@ -54,30 +69,85 @@ def test_login():
             script = polymer_is_trash(['main-page', 'dashboard-view', '#Quick-Links'])
             wait.until(lambda x: x.execute_script(script))
 
-            # Navigate to Jupyter frontend
-            script = polymer_is_trash(
-                ['main-page', 'iframe-link[href=\'/jupyter/?ns=admin\'] .menu-item']
-            )
-            wait.until(lambda x: x.execute_script(script))
-            driver.execute_script(script + '.click()')
-
-            # Click "New Server" button
-            script = polymer_is_trash(['main-page', 'iframe-container', 'iframe'])
-            script += ".contentWindow.document.body.querySelector('#add-nb')"
-            wait.until(lambda x: x.execute_script(script))
-            driver.execute_script(script + '.click()')
-
-            # Enter server name
-            script = polymer_is_trash(['main-page', 'iframe-container', 'iframe'])
-            script += ".contentWindow.document.body.querySelector('#mat-input-0')"
-            wait.until(lambda x: x.execute_script(script))
-            driver.execute_script(script + '.value = "foobar"')
-            driver.execute_script(script + '.dispatchEvent(new Event("input"))')
-
-            # Submit form
-            script = polymer_is_trash(['main-page', 'iframe-container', 'iframe'])
-            script += ".contentWindow.document.body.querySelector('form').submit()"
-            driver.execute_script(script)
+            yield driver, wait, url
         except Exception:
             driver.get_screenshot_as_file('/tmp/selenium.png')
             raise
+
+
+@pytest.mark.full
+@pytest.mark.lite
+def test_login(driver):
+    driver, wait, url = driver
+    driver.get(url)
+
+    # Ensure that we're logged in properly
+    script = polymer_is_trash(['main-page', 'dashboard-view', '#Quick-Links'])
```

Comment:
While I still love this helper function name, with the increased usage and more concise tests, it is becoming less clear what it's actually doing.  It might be worth renaming it to something more descriptive (maybe `select_elements_polymer_fix`) so that other folks working on these tests can tell at a glance what's going on.